packages/react/src/emotion-element.js

```diff
@@ -88,9 +88,7 @@ let Emotion = /* #__PURE__ */ withEmotionCache<any, any>(
     let serialized = serializeStyles(
       registeredStyles,
       undefined,
-      typeof cssProp === 'function' || Array.isArray(cssProp)
```

Comment:
Could you also apply the same fix [here](https://github.com/emotion-js/emotion/blob/dcc72d06ace804330fd285a76c8574f3a89001f9/packages/react/src/global.js#L43-L45)? It would also be sweat if you could add some tests for this fix